### PR TITLE
Added dependabot for creating PR for dependency updates

### LINKED ISSUE #13: Implement dependabot in "build setup"
**Is your feature request related to a problem? Please describe.**
As for the NeIC repos, have automatic PR genereated for new available versions of dependecies

**Describe the solution you'd like**
Adopt the same dependabot setup as the NeIC repos, i.e. neicnordic/sda-pipeline repo.

### PR DESCRIPTION
**Pull request by joshbaskaran**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [x] New feature
- [ ] Unit/integration tests
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change

### Pull request long description:

Added dependabot to the project.

### Related issues:

Fixes #13

### CODE REVIEW

**Comment by joshbaskaran** [comment]
After the merge a maintainer might still have to enable it under setting/security/Code security and analysis.

**Review by kjellp — APPROVED** [review]
A missing newline at the end of dependabot.yml is flagged, might not affect anything, but good to clean up? Will merge now to not make that a blocker.

**Comment by kjellp** [comment]
I followed the instructions to update the settings of the repo, and the effect seems to be to put the same dependabot.yml file into master (updating the file from the this PR merge). I added the newline at the end of the file in that version.

**Pull request merged**
